LifeCycleAssessment_Engine/Compute/EvaluateTM65.cs

```diff
@@ -0,0 +1,155 @@
+            name = name.ToLower();
+
+            double replacementWeightIncrease = 1.1; // is this constant or does this need to be variable? 
+            double bufferFactor = 1.3; // is this constant or does this need to be variable?
+            // Refrigerant to be added later
+            
+            // Define complexity lists
+            // Make this an input 
+            EPDComplexity scaleUpFactor = EPDComplexity.Undefined;
+
+            // Remove lists and add as description to epdComplexity enum
+
+            // Class 1 objects > scaleUpFactor == 1.1
+            List<string> class1 = new List<string>() { 
+                "pipe",
+                "cables",
+                "ducts",
+                "valves",
+                "fire alarm",
+                "access control",
+                "cable containment",
+                "electrical outlets",
+                "busbars"
+            };
+
+            // Class 2 objects > scaleUpFactor == 1.2
+            List<string> class2 = new List<string>() {
+                "pumps",
+                "luminaires",
+                "radiators",
+                "control panels",
+                "lighting control devices",
+                "sensors",
+                "thermal store"
+            };
+
+            // Class 3 objects > scaleUpFactor == 1.3
+            List<string> class3 = new List<string>() {
+                "air handling units",
+                "heat pumps",
+                "boilers",
+                "heat interface units",
+                "chillers",
+                "generator",
+                "MVHR",
+                "switchgear",
+                "UPS"
+            };
+
+            if (class1.Contains(name))
+            {
+                scaleUpFactor = EPDComplexity.Class1;
+            }
+            else if (class2.Contains(name))
+            {
+                scaleUpFactor = EPDComplexity.Class2;
+            }
+            else if (class3.Contains(name))
+            {
+                scaleUpFactor = EPDComplexity.Class3;
+            }
+
+            double scaleFactor = 1;
+
+            switch (scaleUpFactor)
+            {
+                case EPDComplexity.Class1:
+                    scaleFactor = 1.1;
+                    break;
+
+                case EPDComplexity.Class2:
+                    scaleFactor = 1.2;
+                    break;
+
+                case EPDComplexity.Class3:
+                    scaleFactor = 1.3;
+                    break;
+            }
+
+            // This is the calculation 
+            // Need Mass = Density * Volume for all material composition inputs 
+            // Density => All EPDDensity values 
+            // Volume => ???
+
+            // TODO Only use kg based EPDs and remove the Mass query -> give error message if QT is not kg
+
+            double totalQuantity = weight * /*BH.Engine.Matter.Query.Mass(materialComposition)*/ replacementWeightIncrease * scaleFactor * bufferFactor;
```

Comment:
Calculation for material ratios to be multiplied by the object weight specified (mass addition)